kartothek/utils/predicate_converter.py

```diff
@@ -0,0 +1,80 @@
+"""
+Helper module to convert kartothek dataset load predicates into cube conditions.
+"""
+
+from __future__ import absolute_import
```

Comment:
Kartothek is Python 3+ only, so this should not be needed.
```suggestion
```

Reply:
changed